simpleio.py

```diff
@@ -28,15 +28,46 @@
 * Author(s): Scott Shawcroft
 """
 import time
-import sys
-try:
-    import audioio
-except ImportError:
-    pass # not always supported by every board!
 import array
 import digitalio
 import pulseio
 
+#pylint: disable=invalid-name
+def _AudioOut(pin):
+    try:
```

Comment:
i find this construction a little odd, can do we someting ilke `import PWMAudioOut from audiopwmio as AudioOut`?

Reply:
done